web-auth/src/App.tsx

```diff
@@ -1,13 +1,13 @@
-import { makeStyles } from '@mui/styles';
-import { WebAuth } from 'auth0-js';
-import axios from 'axios';
-import React, { useCallback } from 'react';
-import { useGoogleReCaptcha } from 'react-google-recaptcha-v3';
 import LoginForm, {
   Values,
 } from 'web-components/lib/components/form/LoginForm';
 import OnBoardingSection from 'web-components/lib/components/section/OnBoardingSection';
 import { Theme } from 'web-components/lib/theme/muiTheme';
+import { WebAuth } from 'auth0-js';
+import axios from 'axios';
+import React, { useCallback } from 'react';
+import { makeStyles } from 'tss-react/mui';
+import { useGoogleReCaptcha } from 'react-google-recaptcha-v3';
```

Comment:
just a nit but we usually keep the external lib imports above the web-components/ ones